content/java/concepts/iterator/terms/hasPrevious/hasPrevious.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.hasPrevious()'
+Description: 'Returns true if a ListIterator object has prior elements. Returns false otherwise.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Collections'
+  - 'Iterators'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.hasPevious()`** method returns `true` if a `ListIterator` object has prior elements. (i.e. elements before the current element.) Returns `false` otherwise.
```

Comment:
```suggestion
The **`.hasPevious()`** method returns `true` if a `ListIterator` object has prior elements, i.e. elements before the current element. Returns `false` otherwise.
```